src/app/@modal/[...catchAll]/page.tsx

```diff
@@ -0,0 +1,4 @@
+export default function ModalCatchAll() {
```

Comment:
어떤 함수인지는 잘 모르겠지만 테스트 용도라면 삭제 부탁드려요

Reply:
Next.js 공식 문서에서 다른 페이지 이동시 modal를 종료할 때 catch-all route 사용해서 닫을 수 있다고 해서 추가한 함수입니다. 아예 저 function을 지우고 테스트를 해봤는데, route가 제대로 동작이 안되서 유지해놨지만, 아래에 있는 console.log는 지웠습니다.

We use a catch-all route in our @auth slot to close the modal because of the behavior described in [Active state and navigation](https://nextjs.org/docs/app/building-your-application/routing/parallel-routes#active-state-and-navigation). Since client-side navigations to a route that no longer match the slot will remain visible, we need to match the slot to a route that returns null to close the modal.
이 부분이 제가 공식문서에서 찾은 내용입니다.

Reply:
오호 한수 배워갑니다